lib/sda/node.go

```diff
@@ -135,6 +181,19 @@ func (n *Node) EntityList() *EntityList {
 	return n.Tree().EntityList
 }
 
+// List returns the list of TreeNodes cached in the node (creating it if necessary)
+func (n *Node) List() []*TreeNode {
+	if n.treeNodeList == nil {
+		n.treeNodeList = n.Tree().List()
+	}
```

Comment:
I think @nikkolasg wanted to say: from different go routines (intead of structures). But I agree, this should be written in a treadsafe eway. Use RWMutexes (are cheaper) and you write only once, but might read more often (from different go-routines).